mono/tests/Makefile.am

```diff
@@ -684,10 +754,94 @@ PROFILE_DISABLED_TESTS += \
 # in the mobile_static profile
 PROFILE_DISABLED_TESTS += \
 	assembly-load-remap.exe
-else
-PROFILE_DISABLED_TESTS=
+
+PROFILE_DISABLED_TESTS += \
```

Comment:
please add a comment here why they're disabled (yes I know we didn't have it before either, something generic is fine).

I guess we could also move the remoting*.exe to the other instance above which already says `# Tests which rely on remoting`.